### PR TITLE
using new api add Slug scalar

### PR DESCRIPTION
**Pull request by brettski**
**NOTE**: this will not be merged until after thatconference/api package is updated to 1.6.0
Slug scalar verifies slug contains whitelist of characters and aways returns lower case
- Letters, numbers, dash, and underscore
referencing Date scalar now from api
package bumps

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -1 +1 @@
-12.16.2
\ No newline at end of file
+12.16.3
```
Comment (csell5) [inline .node-version:1]: we need to just bump up to 14 here soon.

File: package.json [hunk 2]
```diff
@@ -19,51 +19,51 @@
     "validate": "concurrently npm:test npm:lint npm:build"
   },
   "dependencies": {
-    "@apollo/federation": "^0.13.2",
-    "@google-cloud/firestore": "^3.7.1",
-    "@google-cloud/functions-framework": "^1.5.0",
-    "@sentry/node": "^5.15.0",
+    "@apollo/federation": "^0.16.4",
```
Comment (csell5) [inline package.json:22]: FYI, we need to be careful with this one.. I've had too many breaking changes as x.x. versions have changed. Just check the notes on this one && we need to make sure the other packages are also the same federation version.

File: .node-version (hunk 1 above)
Comment (brettski) [reply .node-version:1]: Understood, but there were specific message warnings when importing packages that they will NOT be compatible with 14 at this time

File: package.json (hunk 2 above)
Comment (brettski) [reply package.json:22]: Suggestion @csell5? How do should we vet this in the future, I am assuming for now, patch level updates should be non-breaking, but minor may be.
The that-api-events package is running 0.16.0, as is Partners and sessions.

**Pull request merged**